quinn-udp/src/unix.rs

```diff
@@ -289,8 +285,7 @@ fn send(
                         // Prevent new transmits from being scheduled using GSO. Existing GSO transmits
                         // may already be in the pipeline, so we need to tolerate additional failures.
                         if state.max_gso_segments() > 1 {
-                            #[cfg(any(feature = "tracing", feature = "direct-log"))]
-                            error!("got transmit error, halting segmentation offload");
+                            crate::log::error!("got transmit error, halting segmentation offload");
```

Comment:
Why import `debug` but qualify `error`?

Reply:
This section has a `cfg` above, namely `#[cfg(target_os = "linux")]`. Importing `error!` would require:

``` diff
+ #[cfg(target_os = "linux")]
+ use super::error;
  use super::{
      cmsg, log::debug, log_sendmsg_error, EcnCodepoint, RecvMeta, Transmit, UdpSockRef,
      IO_ERROR_LOG_INTERVAL,
  };
```

Your call. I am fine with either.

Reply:
That seems like a good reason to me.